libs/sdk-ui-kit/src/Header/HeaderInviteButton.tsx

```diff
@@ -0,0 +1,16 @@
+// (C) 2021-2022 GoodData Corporation
+import React from "react";
+import { useIntl } from "react-intl";
+import { IHeaderInviteButtonProps } from "./typings";
+
+export const HeaderInviteButton: React.FC<IHeaderInviteButtonProps> = ({ onInviteButtonClick }) => {
+    const intl = useIntl();
+    return (
+        <div className="gd-invite-button" onClick={onInviteButtonClick}>
```

Comment:
Should we add `gd-button-small gd-button-primary` here, similar to what we did at https://github.com/gooddata/gooddata-ui-sdk/blob/468fd09e48a7267216aef93c04f2ecde6006d0d9/libs/sdk-ui-kit/src/Header/HeaderUpsellButton.tsx#L9

Reply:
the invite button is not similar css to a button like upsell button (it will similar to gd-header-account and gd-header-help ), so basically I don't think we need to put the css of button here since the style from figma is similar to Help Button and Account button shouldn't it ?

Reply:
ok then.